src/System.Management.Automation/engine/runtime/Operations/ClassOps.cs

```diff
@@ -43,9 +43,26 @@ internal SessionStateKeeper()
 
         internal void RegisterRunspace()
         {
-            // it's not get, but really 'Add' value.
-            // ConditionalWeakTable.Add throw exception, when you are trying to add a value with the same key.
-            _stateMap.GetValue(Runspace.DefaultRunspace, runspace => runspace.ExecutionContext.EngineSessionState);
+            SessionStateInternal ssInMap = null;
```

Comment:
I think it's more of a coding habit. I always make variable assigned before use.